indextools/pds4_create_bundle_member_index.py

```
@@ -12,51 +12,101 @@
 import sys
 
 
-def create_bundle_members(bundle_path, bundle_member_index):
-    """Create a dictionary of LIDs, member status and reference type.
-
-    The path to a directory that contains the bundle.xml file is parsed
-    with lxml.objectify. The bundle member entries are found and scraped
-    for their LID. This LID is put into the bundle_member_lid dictionary
-    as a key with the reference type and the member status scraped and
-    entered as values. The input 'bundle_path' is the path to the bundle.
-    The input bundle_member_index is the dictionary that is populated with
-    the bundle member information.
+def get_bundle_filepaths(bundle_path):
+    """Find and store path to the bundle.xml file in the bundle.
+    
+    In the event a path to the bundle.xml file is not found, a terminal
```

Just a comment going forward...since you're planning to make these functions generic and useable by a variety of other programs, I strongly suggest you get rid of anything that prints a message or, especially, exits. A library function should never exit the program. Instead you should throw an exception (with an appropriate message), and then the calling functions should catch the exception and decide what to do with it - print the message and exit, log the message and skip that bundle, whatever. But it should be up to the caller.

---

You make a good point that we eventually want to implement better error-handling.  I've created Issue #47 to remind us to do this.